packages/reference/src/components/MissingEntityCard/MissingEntityCard.tsx

```diff
@@ -17,8 +17,7 @@ export function MissingEntityCard(props: {
       <Flex alignItems="center" justifyContent="space-between">
         <div className={props.asSquare ? styles.squareCard : ''}>
           <SectionHeading marginBottom="none">
-            {props.entityType === 'Entry' && 'Entry is missing or inaccessible'}
-            {props.entityType === 'Asset' && 'Asset is missing or inaccessible'}
+            {`{props.entityType} is missing or inaccessible`}
```

Comment:
ops, I think you accepted the suggestion before i added the correct string interpolation `$`, sorry :'(

Reply:
no worries, I also noticed. Will push a quick commit after I confirm that there is no issues

Reply:
now it should be fine